content/backstage/plugins/jenkins.md

```diff
@@ -12,54 +12,85 @@ seo:
     Build, test, and deploy on Jenkins CI/CD platform.
 
 logoImage: '../../assets/logos/jenkins/logo-jenkins.png'
-coverImage: '../../assets/jenkins-plugin.png'
+coverImage: '../../assets/backstage/plugins/jenkins/jenkins-plugin.png'
 coverImageAlt: 'A screenshot of the Jenkins plugin.'
 
 gettingStarted:
-  - intro: Install the plugin into Backstage.
+  - intro: Install the plugin.
     language: bash
-    code: 'yarn add @backstage/plugin-jenkins'
-
-  - intro: Add plugin to the list of plugins.
-    language: typescript
     code: |
-      // packages/app/src/plugins.ts
-      export { plugin as Jenkins } from '@backstage/plugin-jenkins';
+      cd packages/app
+      yarn add @backstage/plugin-jenkins
 
-  - intro: Add plugin API to your Backstage instance.
+  - intro: Add the plugin API to your Backstage app.
     language: typescript
     code: |
       // packages/app/src/components/catalog/EntityPage.tsx
-      import {
-        Router as JenkinsRouter,
-        isPluginApplicableToEntity as isJenkinsAvailable,
-        LatestRunCard as JenkinsLatestRunCard,
-      } from '@backstage/plugin-cloudbuild';
-
-      const CICDSwitcher = ({ entity }: { entity: Entity }) => {
-        switch (true) {
+      import { EntityJenkinsContent } from '@backstage/plugin-jenkins';
+
+      // add new Jenkins tab to the service component page
+      const serviceEntityPage = (
+        <EntityLayout>
           ...
-          case isJenkinsAvailable(entity):
-            return <JenkinsRouter entity={entity} />;
+          <EntityLayout.Route path="/jenkins" title="Jenkins">
+            <EntityJenkinsContent />
+          </EntityLayout.Route>
           ...
-        }
-      }
+        </EntityLayout>
+      );
 
-  - intro: Add proxy configuration
+  - intro: Add proxy configuration to your Backstage app's config. A Jenkins API token is required.
```

Comment:
It's worth mentioning that there are instructions which explain how to get the token further down the page. You can even use markdown to link to the relevant section.